extensions/data-workspace/package.json

```diff
@@ -71,6 +71,10 @@
       {
         "command": "projects.removeProject",
         "title": "%remove-project-command%"
+      },
+      {
+        "command": "projects.manageProject",
```

Comment:
wasn't this already added in the initial dashboard PR?

Reply:
Yes, messed up that branch somehow. Fixed it now.